### DISCUSSION TITLE
Receiving error: missing required module 'CwlCatchExceptionSupport'

### DISCUSSION THREAD

**Discussion by brandtdaniels** [Quick/Nimble#992] [Q&A] [answered]
I'm adding Quick & Nimble (via Cocoapods) to large project that has been around for a long time and has mostly used XCTest. I'm attempting to add the first QuickSpec, and receiving error: `missing required module 'CwlCatchExceptionSupport'` when trying to run the test/spec

```
    pod 'Nimble', '~> 10.0.0'
    pod 'Quick', '~> 5.0.0'
```

**Comment by younata** [comment]
Hey @brandtdaniels, thanks for reaching out!

As an initial reaction to this, I'm a little stumped. This should work. We ship `CwlCatchExceptionSupport` as part of Nimble (especially in the cocoapod version), so there shouldn't be a missing module.

The only thing I can think of is that watchOS (which isn't supported in 10.0.0 anyway) and tvOS both specifically don't include `CwlCatchExceptionSupport` because it doesn't work with those platforms. But we have compiler directives in `throwAssertion` to avoid that, and we've been shipping tvOS support for long enough that I'm surprised this is the first we're hearing of issues with it. Unless the issue is we need to do something similar in `raiseException` or somewhere else?

What platforms are you trying to test for? I assume just iOS, but anything else? Can you provide other details that might help us to narrow this down?

**Comment by ColdLogical** [comment, marked as answer]
Had this problem when I created a second test target, the messages here pointed me to realize that Quick and Nimble had not been added in the `Link Binary with Libraries` section of `Build Phases` on my new test target.
